code/js/lib/jquery.i18n/jquery.i18n.emitter.js

```diff
@@ -0,0 +1,168 @@
+/*!
```

Comment:
Renamed folder lib to ext-libs
It is developed by external so it must be clear what is external.

Reply:
Sorry. For folders/packages we will use singular. so just ext-lib  ... my mistake.

Reply:
no problem... it is correct now... ext-lib

Reply:
ok, but you push it .. I can't see it

Reply:
I am sorry, pushed... all done hopefully :0